src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -699,6 +784,7 @@ function tryRemoveElement(elem) {
         }
 
         destroy() {
+            destroyShakaPlayer(this);
```

Comment:
Ideally, make `destroy` async and await `destroyShakaPlayer` to eliminate possible race. But that would require to await `destroy` elsewhere and checking other players (Android, JMP, etc.). So I dunno if we should do it right now.